infra/charts/turing/templates/cluster-role-binding.yaml

```diff
@@ -2,13 +2,13 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: {{ template "turing.service-account" . }}
```

Comment:
I'm curious, did you encounter the problem with trying to get the hyphenated version of variable names to work or did you do this out of consistency? I ran into this problem in the past (https://github.com/helm/helm/issues/2192)